util/dvsim/FlowCfg.py

```diff
@@ -87,19 +87,18 @@ def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
         self.results_title = ""
         self.revision = ""
         self.results_server_prefix = ""
-        self.results_server_url_prefix = ""
         self.results_server_cmd = ""
         self.css_file = os.path.join(
             os.path.dirname(os.path.realpath(__file__)), "style.css")
-        # `self.results_path` will be updated after `self.rel_path` and
+        # `self.results_*` below will be updated after `self.rel_path` and
         # `self.scratch_base_root` variables are updated.
-        self.results_path = ""
+        self.results_local_dir = ""
+        self.results_local_page = ""
         self.results_server_path = ""
         self.results_server_dir = ""
-        self.results_server_html = ""
         self.results_server_page = ""
-        self.results_summary_server_html = ""
-        self.results_summary_server_page = ""
+        self.results_server_url = ""
+        self.results_html_name = ""
```

Comment:
I preserved several variables here since otherwise even more refactoring would be required in `def _publish_results(self):`.